streams/src/test/java/org/apache/kafka/streams/processor/internals/TaskManagerTest.java

```diff
@@ -542,6 +552,39 @@ public void shouldPropagateExceptionFromStandbyCommit() {
         verify(standby);
     }
 
+    @Test
+    public void shouldSendPurgeData() {
+        final KafkaFuture<DeletedRecords> futureDeletedRecords = new KafkaFutureImpl<>();
+        final Map<TopicPartition, RecordsToDelete> recordsToDelete = Collections.singletonMap(t1p1, RecordsToDelete.beforeOffset(5L));
+        final DeleteRecordsResult deleteRecordsResult = new DeleteRecordsResult(Collections.singletonMap(t1p1, futureDeletedRecords));
+
+        EasyMock.expect(active.recordsToDelete()).andReturn(recordsToDelete).once();
+        EasyMock.expect(adminClient.deleteRecords(recordsToDelete)).andReturn(deleteRecordsResult).once();
+        replay();
```

Comment:
Do we not need to pass in `active` and `adminClient` here?

Reply:
`replay()` will replay all the mock clients.